Source/WebKit/Shared/Cocoa/WebCoreArgumentCodersCocoa.mm

```diff
@@ -607,9 +627,14 @@
     encoder << info.elementBounds;
 }
 
+template<> Class getClass<DDScannerResult>()
```

Comment:
It might be nice to put all these overloads together.

Reply:
I did it this way because each of them was in an #if ENABLE() block.

Will go lump all the PK ones together, though.